sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/InsertIntoHadoopFsRelationCommand.scala

```diff
@@ -132,6 +132,9 @@ case class InsertIntoHadoopFsRelationCommand(
             }
           }
         }
+
+        sparkSession.catalog.refreshByPath(outputPath.toString)
```

Comment:
why? we will do `fileIndex.foreach(_.refresh())` at the end, what's the difference between this and `refreshByPath`?